src/ch/imedias/rsccfx/model/Rscc.java

```diff
@@ -1,61 +1,67 @@
 package ch.imedias.rsccfx.model;
 
-import java.io.BufferedReader;
-import java.io.File;
-import java.io.InputStreamReader;
+import javafx.beans.property.SimpleStringProperty;
+import javafx.beans.property.StringProperty;
 
 public class Rscc {
   /**
    * Points to the "docker-build_p2p" folder inside resources, relative to the build path.
    * Important: Make sure to NOT include a / in the beginning or the end.
    */
-  private static final String PATH_TO_RESOURCE_DOCKER = "resources/docker-build_p2p";
+  private final String pathToResourceDocker;
 
-  private final SystemCommander systemCommander;
+  private final SystemCommander systemCommander; // FIXME: try it with ProcessExecutor
 
+  private final StringProperty key = new SimpleStringProperty();
+
+  /**
+   * Initializes the Rscc model class.
+   */
   public Rscc(SystemCommander systemCommander) {
+
     this.systemCommander = systemCommander;
+    pathToResourceDocker = getClass().getClassLoader().getResource("docker-build_p2p")
+        .getFile().toString().replaceFirst("file:","");
   }
 
   /**
    * Kills the connection to the keyserver.
-   * @param key generated by requestKey().
+   * @param key generated by requestKey(). // FIXME: there is no requestKey method?
```

Comment:
correct, but the method is called "requestTokenFromServer", so it should be corrected to that :)